fluss-flink/fluss-flink-common/src/main/java/com/alibaba/fluss/flink/row/RowConverters.java

```diff
@@ -0,0 +1,172 @@
+ * storage system.
+ */
+public class RowConverters {
+    /**
+     * Converts a POJO to a Fluss GenericRow. Special handling is provided for timestamp types.
+     *
+     * @param pojo The POJO to convert
+     * @return A GenericRow containing the POJO's field values
+     * @throws IllegalArgumentException if there's an error accessing the fields
+     */
+    public static GenericRow pojoToGenericRow(Object pojo) {
+        if (pojo == null) {
+            throw new IllegalArgumentException("Input POJO cannot be null");
+        }
+
+        Field[] allFields = pojo.getClass().getDeclaredFields();
+
+        // Filter out synthetic fields (JaCoCo's $jacocoData)
+        Field[] fields =
+                Arrays.stream(allFields)
+                        .filter(
+                                field ->
+                                        !field.isSynthetic()
+                                                && !Modifier.isStatic(field.getModifiers()))
+                        .toArray(Field[]::new);
+
+        Object[] values = new Object[fields.length];
+
+        for (int i = 0; i < fields.length; i++) {
+            Field field = fields[i];
+            field.setAccessible(true);
+            try {
+                Object value = field.get(pojo);
+
+                // Handle various field types
+                if (value instanceof String) {
+                    values[i] = BinaryString.fromString((String) value);
+                } else if (value instanceof LocalDateTime) {
+                    values[i] = TimestampNtz.fromLocalDateTime((LocalDateTime) value);
+                } else if (value instanceof Date) {
+                    values[i] = TimestampNtz.fromMillis(((Date) value).getTime());
+                } else if (value instanceof Instant) {
+                    LocalDateTime ldt =
+                            LocalDateTime.ofInstant((Instant) value, ZoneId.systemDefault());
+                    values[i] = TimestampNtz.fromLocalDateTime(ldt);
+                } else {
+                    values[i] = value;
+                }
+            } catch (IllegalAccessException e) {
+                throw new IllegalArgumentException("Unable to access field: " + field.getName(), e);
+            }
+        }
+
+        return row(values);
+    }
+
+    public static GenericRow row(Object... objects) {
+        GenericRow row = new GenericRow(objects.length);
+        for (int i = 0; i < objects.length; i++) {
+            if (objects[i] instanceof String) {
+                row.setField(i, BinaryString.fromString((String) objects[i]));
+            } else {
+                row.setField(i, objects[i]);
+            }
+        }
+        return row;
+    }
+
+    /**
+     * Converts a Fluss RowType to a Flink RowType.
+     *
+     * @param flussRowType The Fluss RowType to convert
+     * @return The equivalent Flink RowType
+     */
+    public static org.apache.flink.table.types.logical.RowType flussRowTypeToFlinkRowType(
```

Comment:
`com.alibaba.fluss.flink.utils.FlinkConversions.toFlinkRowType` already covers this.